README.rst

```diff
@@ -1038,6 +1038,25 @@ Choose from the list of available rules:
     ``'newest'``
   - ``use_class_const`` (``bool``): use ::class notation; defaults to ``true``
 
+* **php_unit_short_will_return**
+
+  Usage of ``->will($this->returnValue(..))`` statements must be replaced by
+  it's shorter equivalent ``->willReturn(...)``.
+
+  *Risky rule: risky when PHPUnit classes are overridden or not accessible, or when project has PHPUnit incompatibilities.*
+
+  Configuration options:
+
+  - ``returnArgument`` (``false``, ``true``): fix ``returnArgument`` occurrence; defaults
```

Comment:
why would one want to fix `returnArgument`, yet not `returnCallback` ?

Reply:
I assume that someone could have some of methods overwritten or used in some non-standard way that fixer will break the final output so with this configuration it will be possible to skip such cases. By default it's turned on so IMO there're no contradictions.

Reply:
even if enabled by default, I still see it as adding a complexity to the fixer and could create same question for end-user.
is this a real case when user do such a things? I would rather call it big misuse of phpunit which we shall not support.

what do you think ?

Reply:
I just thought it could be nice to fix only selected methods not all bundled together - I will not stick to my configurable solution and can remove this part as you wish ;)